packages/devtools/lib/src/ui/dragscroll.dart

```diff
@@ -0,0 +1,94 @@
+import 'dart:html';
+
+class DragScroll {
+  var mousemove = 'mousemove';
+  var mouseup = 'mouseup';
+  var mousedown = 'mousedown';
+  var EventListener = 'EventListener';
+
+// ??? what are these ???
+//  var addEventListener = 'add'+EventListener;
+//  var removeEventListener = 'remove'+EventListener;
+
+
+
+  num newScrollX;
+  num newScrollY;
+
+  var dragged = [];
+
+  void reset(int i, /*type?*/ eventListener) {
```

Comment:
I think understanding what all these types should be would help. What type should eventListener be?

Reply:
Element